.vscode/extensions.json

```diff
@@ -11,7 +11,13 @@
 		// Formatting
 		"esbenp.prettier-vscode",
 		"stkb.rewrap",
+		"eriklynd.json-tools",
+		"DotJoshJohnson.xml",
 		// Testing
-		"spoonscen.es6-mocha-snippets"
+		"spoonscen.es6-mocha-snippets",
+		// Documentation
+		"ms-devlabs.extension-manifest-editor",
+		"streetsidesoftware.code-spell-checker",
+		"hnw.vscode-auto-open-markdown-preview"
```

Comment:
I'd submitted this PR a while back because someone (don't remember who) had asked me to submit a PR with the extensions I was using, but if these aren't useful to other people there's no need to add them. I'm fine with closing this PR without merging it if that's the case.

Reply:
@vazexqi thoughts?

Reply:
Let's close this one. I don't use these enough to recommend them.

Reply:
Done.